CHANGELOG.md

```diff
@@ -447,6 +447,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Add document management
 - Cache database queries and serialization in LTI views for students
+- Add a beta label on live configuration buttons
```

Comment:
I didn't noticed but why did you put this in release `3.0.0` ? 😂
I will move it in the `unreleased` one